include/ghex/rma/thread/handle.hpp

```diff
@@ -0,0 +1,65 @@
+/* 
+ * GridTools
+ * 
+ * Copyright (c) 2014-2020, ETH Zurich
+ * All rights reserved.
+ * 
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * 
+ */
+#ifndef INCLUDED_GHEX_RMA_THREAD_HANDLE_HPP
+#define INCLUDED_GHEX_RMA_THREAD_HANDLE_HPP
+
+#include "../locality.hpp"
+
+namespace gridtools {
+namespace ghex {
+namespace rma {
+namespace thread {
+
+struct info
+{
+    void* m_ptr;
+};
+
+struct local_data_holder
```

Comment:
These classes are small and quite clear to me. It would be cool to have an overview of the strategy somewhere, but I do not know where

Reply:
I can add comments of the overall strategy in the bulk_communication_object.hpp - seems the most top level point.